model/src/main/java/org/cloudfoundry/identity/uaa/zone/IdentityZoneConfiguration.java

```diff
@@ -24,6 +25,7 @@
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class IdentityZoneConfiguration {
 
+    private LockoutPolicy clientLockoutPolicy = new LockoutPolicy();
```

Comment:
Like tokenPolicy, clientLockoutPolicy defaults empty values to -1. The LockoutPolicyRetriever returns the global policy in getLockoutPolicy() when it finds an empty lockout policy in the identity zone config, so there is a fallback.